packages/protocol-http/package.json

```diff
@@ -16,6 +16,13 @@
   "main": "./dist-cjs/index.js",
   "module": "./dist-es/index.js",
   "types": "./dist-types/index.d.ts",
+  "browser": {
+    "./dist-es/httpHandlerInitialization": "./dist-es/httpHandlerInitialization.browser"
+  },
```

Comment:
does this need a `dist-cjs` entry point?

Reply:
no, I'm going to remove this